monkey/infection_monkey/exploit/tests/zerologon_utils/test_vuln_assessment.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+from nmb.NetBIOS import NetBIOS
+
+from infection_monkey.exploit.zerologon_utils.vuln_assessment import \
+    get_dc_details
+from infection_monkey.model.host import VictimHost
+
+
+DOMAIN_NAME = "domain-name"
+IP = "0.0.0.0"
+
+
+@pytest.fixture
+def host():
+    return VictimHost(IP, DOMAIN_NAME)
+
+
+def test_get_dc_details_multiple_netbios_names(host, monkeypatch):
+    def mock_queryIPForName(*args, **kwargs):
+        return NETBIOS_NAMES
+
+    monkeypatch.setattr(NetBIOS, "queryIPForName", mock_queryIPForName)
+
+    NETBIOS_NAMES = ["Name1", "Name2", "Name3"]
+    dc_ip, dc_name, dc_handle = get_dc_details(host)
+    assert dc_ip == IP
+    assert dc_name == NETBIOS_NAMES[0]
+    assert dc_handle == f"\\\\{NETBIOS_NAMES[0]}"
+
+
+def test_get_dc_details_no_netbios_names(host, monkeypatch):
+    def mock_queryIPForName(*args, **kwargs):
+        return NETBIOS_NAMES
+
+    monkeypatch.setattr(NetBIOS, "queryIPForName", mock_queryIPForName)
+
```

Comment:
What if we create a function that accepts `NETBIOS_NAMES`  as a parameter and patches `NetBIOS` appropriately? Then this code wont be duplicated in 2 tests.